atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/todo/TodoClient.java

```diff
@@ -159,4 +179,41 @@ SweepResults sweepTable(TableReference table) {
                 .build();
         return sweepTaskRunner.get().run(table, sweepConfig, PtBytes.EMPTY_BYTE_ARRAY);
     }
+
+    public long numCellsDeleted(TableReference tableRef) {
+        return cellsDeleted(tableRef).size();
+    }
+
+    public long numCellsDeletedAndSwept(TableReference tableRef) {
+        return cellsDeleted(tableRef).entrySet().stream()
+                .filter(entry -> getValueForEntry(tableRef, entry).getTimestamp() == Value.INVALID_VALUE_TIMESTAMP)
+                .count();
+    }
+
+    private Map<Cell, Value> cellsDeleted(TableReference tableRef) {
```

Comment:
The name sounds like it's counting the number of cells that have been deleted (i.e. swept), which is wrong, and makes "numCellsDeletedAndSwept" confusing.

You're actually getting the atlas deletes, so how about `getAllAtlasDeletes(tableRef)` (and `numAtlasDeletes` and `numSweptAtlasDeletes` above)?